### DISCUSSION TITLE
Strange "SELECT ALL" queries

### DISCUSSION THREAD

**Discussion by Illia-M** [gramps-project/gramps-web-api#573] [General]
I see queries like `SELECT handle, blob_data FROM note` when I open Notes for Source and other queries like  `SELECT handle, blob_data FROM media` in other places(don't know which page trigger that), but it timeout on my tree with 28k+ Notes.

Should some queries be more "smart" and delegate filter logic to DB?

**Comment by DavidMStraub** [comment]
Converting to a discussion.

Note that there's a lot to be said about performance and ways to improve it. See e.g. this thread: https://gramps.discourse.group/t/how-to-make-gramps-web-fast/6266
